packages/fx-core/src/component/driver/aad/utility/aadAppClient.ts

```diff
@@ -53,25 +55,38 @@ export class AadAppClient {
   @hooks([ErrorContextMW({ source: "Graph", component: "AadAppClient" })])
   public async createAadApp(
     displayName: string,
-    signInAudience = SignInAudience.AzureADMyOrg
+    signInAudience = SignInAudience.AzureADMyOrg,
+    logProvider?: LogProvider
```

Comment:
can we pass the log provider in constructor? So we don't need to update every function in this class.